zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java

```diff
@@ -87,21 +83,12 @@ public void addViewControllers(ViewControllerRegistry registry) {
     registry.addRedirectViewController("/info", "/actuator/info");
   }
 
-  @Bean ArmeriaServerConfigurator corsConfigurator(
+  /** Configures the server at the last because of the specified {@link Order} annotation. */
+  @Order @Bean ArmeriaServerConfigurator corsConfigurator(
```

Comment:
Does this need to be run after the other catch all bean? If the two have an ordering dependencies, we should specify values to `@Order` to guarantee the order I think.

Reply:
Yes, you're right. But the orders for the other `ArmeriaServerConfigurator` beans are `0` which is a default when no `@Order` annotation is specified. And just specifying `@Order` makes its bean lowest precedence because the annotation's default value is `Integer.MAX_VALUE` (which means lowest precedence). 

So it's okay to specify `@Order` only from here. But if you want to make their order clear, it would also be okay to specify order on all `ArmeriaServerConfigurator` beans. :-)

See: 
- https://github.com/line/armeria/pull/1535
- https://line.github.io/armeria/apidocs/com/linecorp/armeria/spring/ArmeriaServerConfigurator.html#getOrder()

Reply:
Ah no don't think we need to always specify it when there is no order dependency for the default value. But just wanted to confirm there is no ordering issue with the bean here which has the same order.

https://github.com/openzipkin/zipkin/blob/master/zipkin-autoconfigure/metrics-prometheus/src/main/java/zipkin2/autoconfigure/prometheus/ZipkinPrometheusMetricsAutoConfiguration.java

Reply:
Ah, got it. I missed that. I guess there's no ordering issue between them, but I will change their order in order to make it clear. Thank you!